Assets/Scripts/Card/CardCreator.cs

```diff
@@ -25,4 +26,13 @@ public GameObject createCard(Card card, Transform parent, Action<SimpleCard> _cl
         newCard.GetComponent<SimpleCard>().Setup(facesPrefabsDict[card.Name], _clickAction, card.guid);
         return newCard;
     }
+
+    public GameObject getOrCreateCard(Dictionary<Guid, GameObject> cardsInScene, Card card, Transform parent, Action<SimpleCard> _clickAction)
```

Comment:
As far as I remember we have agreed upon that we won't use any type of conjunctions in methods' names. From caller POV it is important to get card with no additional details of the implementation (either card is already there or not) so action u're trying is achieve just get the card. So I see 2 possible options here:

1) Instead of using conjunction create methods that will create some sort of chain like: `doesExist` -> `getCard` (if so) -> `createCard` (if it doesn't).
2) Just name it as `getCard` and hide all implementation inside with no additional information if card will be created in case it doesn't exist yet. 

Personally I'd rather stay with second option.

Reply:
good point, I will rename it to `getCard`